pkgs/tools/system/nvitop/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, stdenv
+, pkgs
+, python3Packages
+, fetchFromGitHub
+, makeWrapper
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nvitop";
+  version = "0.10.0";
+
+  src = fetchFromGitHub {
+    owner = "XuehaiPan";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-nGdEMLxpw2Ts0dypkoZg3r2NF4IeT1ykbRmrmf9qxrA=";
+  };
+
+  nativeBuildInputs = [ ];
+
+  propagatedBuildInputs = with python3Packages; [
+    cachetools
+    psutil
+    termcolor
+    nvidia-ml-py
+  ];
+
+  doCheck = false;
```

```suggestion
  # has no tests
  doCheck = false;
```

and missing pythonImportsCheck if that works.

---

This is an application, so I guess, an import test does not really makes sense.
Maybe more:
```
checkPhase = ''
  nvitop --version
'';
```
Does this make sense ?

---

We can do that, too.

---

Ok, I removed `doCheck = false;` and added this !